x/pairing/keeper/cu_tracker_test.go

```diff
@@ -249,18 +249,21 @@ func TestTrackedCuWithQos(t *testing.T) {
 			}
 			ts.relayPaymentWithoutPay(relayPaymentMessage2, true)
 
-			balance1 := ts.GetBalance(provider1Acc.Addr)
-			balance2 := ts.GetBalance(provider2Acc.Addr)
-
 			// advance month + blocksToSave + 1 to trigger the provider monthly payment
 			ts.AdvanceMonths(1)
 			ts.AdvanceBlocks(ts.BlocksToSave() + 1)
 
-			newBalance1 := ts.GetBalance(provider1Acc.Addr)
-			newBalance2 := ts.GetBalance(provider2Acc.Addr)
-
-			require.Equal(t, balance1+tt.p1ExpectedReward, newBalance1)
-			require.Equal(t, balance2+tt.p2ExpectedReward, newBalance2)
+			reward, err := ts.QueryDualstakingDelegatorRewards(provider1Acc.Addr.String(), provider1Acc.Addr.String(), ts.spec.Index)
+			require.Nil(ts.T, err)
+			require.Equal(ts.T, tt.p1ExpectedReward, reward.Rewards[0].Amount.Amount.Int64())
+			_, err = ts.TxDualstakingClaimRewards(provider1Acc.Addr.String(), provider1Acc.Addr.String())
```

Comment:
why not checking the balance after claiming? also verify that the rewards are reset after claiming?